docs/installation.rst

```diff
@@ -0,0 +1,188 @@
+Installation
+------------
+There are two options, either install all the dependencies yourself or setup the engine
+within a  Docker image. Whatever option you choose, you will need an external solver to
```

Comment:
It seems to have two spaces before `Docker`?

Reply:
Done